frontend/src/bundles/common/components/widgets/activity-widget/activity-widget.tsx

```diff
@@ -0,0 +1,75 @@
+import { type ReactNode } from 'react';
+
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+type Properties = {
+    label: string;
+    value: string;
+    color: ValueOf<typeof ActivityWidgetColor>;
+    icon: ReactNode;
+};
+
+const ActivityWidgetColor = {
+    YELLOW: 'yellow',
+    MAGENTA: 'magenta',
+    PURPLE: 'purple',
+} as const;
+
+const ActivityWidget = ({
+    label,
+    value,
+    color,
+    icon,
+}: Properties): JSX.Element => {
+    const classes = {
+        base: 'w-[15.625rem] h-[10.5rem] rounded-[0.5rem] text-white p-[1rem] flex gap-[1rem] bg-no-repeat bg-bottom',
```

Comment:
If for example there would be not 3 widgets, in this case I think we should avoid hardcoded width, or just provide optional property className maybe. (or there would be definitely 3 widgets?)